Src/FluentAssertions/Events/EventMonitor.cs

```diff
@@ -19,18 +19,21 @@ internal class EventMonitor<T> : IMonitor<T>
     private readonly ConcurrentDictionary<string, EventRecorder> recorderMap =
         new ConcurrentDictionary<string, EventRecorder>();
 
-    public EventMonitor(object eventSource, Func<DateTime> utcNow)
+    public EventMonitor(object eventSource, EventMonitorOptions options)
     {
         Guard.ThrowIfArgumentIsNull(eventSource, nameof(eventSource), "Cannot monitor the events of a <null> object.");
 
+        this.options = options ?? new EventMonitorOptions();
```

Comment:
Is there a real use-case for accepting a null `options`?

Reply:
fixed